### PR TITLE
feat: added chat layout component

### PR DESCRIPTION
**Pull request by elpollo12**
### Description

Added chat layout with styled components

fixes # (issue)

#34 

#### Screenshots
scrnli_22_10_2021 10-47-59

### CODE REVIEW

File: src/pages/Chat.jsx [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+  ChatLayoutContainer,
+  HeaderChat,
+  HeaderTitle,
+  HeaderText,
+  FooterChat,
+  FooterChatInput,
+  ChatBodyWrapper,
+  HeaderChatImg,
+} from '../layouts/Chat.styled'
+import { BackBtn } from '../layouts/Map.styled'
```
Comment (Israel-Laguan) [inline src/pages/Chat.jsx]: Move this to components/Button please

**Comment by Israel-Laguan** [comment]
I tested and header looks kinda big in mobile view
image

Please add a media query for mobile

**Comment by elpollo12** [comment]
Header height reduced in mobile view @Israel-Laguan

File: src/layouts/Chat.styled.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+`
+
+const HeaderChat = styled.div`
+  display: flex;
+  flex-direction: row;
+  background: var(--primary);
+  border-bottom-right-radius: 25px;
+  border-bottom-left-radius: 25px;
+  padding: 1.5%;
+  @media (min-width: 960px) {
```
Comment (Israel-Laguan) [inline src/layouts/Chat.styled.jsx:19]:
```suggestion

  @media (min-width: 960px) {
```

File: src/layouts/Chat.styled.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+  @media (max-width: 960px) {
+    height: 11vh;
+  }
+`
+
+const BackBtn = styled.img`
+  @media (max-width: 960px) {
+    height: 100%;
+  }
```
Comment (Israel-Laguan) [inline src/layouts/Chat.styled.jsx]:
```suggestion
 height: 100%;
```

File: src/layouts/Chat.styled.jsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    height: 11vh;
+  }
+`
+
+const BackBtn = styled.img`
+ height: 100%;
+
+  @media (min-width: 961px) {
+    height: 100%;
+  }
```
Comment (Israel-Laguan) [inline src/layouts/Chat.styled.jsx]:
```suggestion
```

File: src/layouts/Chat.styled.jsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  flex: ${props => props.cols};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  @media (max-width: 960px) {
+    padding: 5% 0;
+  }
```
Comment (Israel-Laguan) [inline src/layouts/Chat.styled.jsx]:
```suggestion
  padding: 5% 0;
```

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**